must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -217,7 +217,7 @@ for ns in $namespaces; do
     # creating a counter variable for collecting PID in array
     idx=0
     # Collecting ceph crash dump
-    for node in $(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/Ready/{print $1}'); do
+    for node in $(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers |awk '/\yReady\y/{print $1}'); do
```

Comment:
Add back the whitespace removed here.

Reply:
Actually, since this and the previous for conditional are now the same, could you perform the check once and save the results into an array? Then you can just loop over the array and save a call to `oc get`.

Reply:
this has been done